cmd/vinegar/app.go

```diff
@@ -203,8 +205,8 @@ func (a *app) showError(e error) {
 	// In a bootstrapper context, the window is destroyed to show the
 	// error instead, which will make the GtkApplication exit.
 	a.Hold()
-	d := adw.NewAlertDialog("Something went wrong", e.Error())
-	d.AddResponses("okay", "Ok", "open", "Open Log")
+	d := adw.NewAlertDialog(L("Something went wrong"), L(e.Error()))
+	d.AddResponses("okay", L("Ok"), "open", L("Open Log"))
```

Comment:
Sorry! Errors are not to be translated, as it could be *any* value, which is not addressed in the actual locale translation.